posts/solving-parallel-letter-frequency-elixir-part-2-adding-concurrency.md

```diff
@@ -1,6 +1,8 @@
-# "Solving Parallel Letter Frequency in Elixir - Part II: Adding Concurrency with `Task`"
+# 2. What Parallel Letter Frequency can teach you about concurrency with Elixir’s `Task` module
 
-In `Part I` I showed how I applied Unicode matching in Elixir to help me solve the first part of [Parallel Letter Frequency](https://exercism.io/tracks/elixir/exercises/parallel-letter-frequency), a medium difficulty problem on [Exercism's Elixir Track](https://exercism.io/tracks/elixir).
+Exercises on Exercism are small, synthetic, and often seemingly trivial. It’s easy to imagine that experienced practitioners would have nothing to learn from them. However, solving these synthetic problems can push you to learn and apply parts of your language that you may not have explored. This new learning can lead you to solve real world problems more efficiently or in a more expressive way.
+
+[Parallel Letter Frequency](https://exercism.io/tracks/elixir/exercises/parallel-letter-frequency) is a medium difficulty exercise on [Exercism's Elixir Track](https://exercism.io/tracks/elixir). It asks you to write a function that calculates the frequency of letters in a list of strings, and to do so concurrently. This unpacks a surprising number of interesting lessons.
```

Comment:
Here too we need a transition from the intro, in order to introduce the problem that this blog post is talking about.